apps/contestedroll.js

```diff
@@ -217,7 +217,7 @@ export class ContestedRoll {
             if (rollmode == 'gmroll' && !game.user.isGM)
                 whisper.push(game.user.id);
 
-            if (game.dice3d != undefined && roll instanceof Roll) { // && !fastForward) {
+            if (game.dice3d != undefined && roll instanceof Roll && game.system.id != 'swade') { // && !fastForward) {

```

Comment:
It's likely that both of these "roll 3d dice if 3d dice" checks belong in their own function for clarity's sake but's kinda a personal preference thing.